test/parallel/test-http2-session-settings.js

```diff
@@ -104,15 +104,15 @@ server.listen(
         ['maxHeaderListSize', 2 ** 32],
         ['maxHeaderSize', -1],
         ['maxHeaderSize', 2 ** 32],
-      ].forEach((i) => {
+      ].forEach(([key, value]) => {
```

Comment:
This is fine but it would probably be worth changing this for a `for (...)` loop instead of `forEach(...)`